src/main/resources/init.sql

```diff
@@ -0,0 +1,15 @@
+DROP SCHEMA IF EXISTS CAESAR;
+CREATE SCHEMA CAESAR CHARACTER SET UTF8;
+USE CAESAR;
+
+
+CREATE USER IF NOT EXISTS 'admin'@'localhost' IDENTIFIED BY '1234%';
+GRANT ALL PRIVILEGES ON *.* TO 'admin'@'localhost';
+
+CREATE USER IF NOT EXISTS 'db_admin'@'localhost' IDENTIFIED BY '1234%';
+GRANT ALL PRIVILEGES ON CAESAR.* TO 'admin'@'localhost';
+
+CREATE USER IF NOT EXISTS 'user_ro'@'localhost' IDENTIFIED BY '1234%';
+GRANT SELECT ON CAESAR.* TO 'user_ro'@'localhost';
```

Comment:
Please advance this password a bit beyond 1234 and place it in readme, under some new section like 'how to read the data from db'.